### PR TITLE
Add component command

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

**Comment by pablogs9** [comment]
@mergify backport main

File: scripts/component [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        "url" : "https://github.com/micro-ROS/micro_ros_nuttx_app"
+    },
+    {
+        "name": "azure_rtos",
+        "description": "micro-ROS module for Microsoft Azure RTOS: this package enables the integration of micro-ROS in a Microsoft Azure RTOS workspace.",
+        "url" : "https://github.com/micro-ROS/micro_ros_azure_rtos_app"
+    },
+    {
+        "name": "tiva_c_series",
+        "description": "micro-ROS app for TI Tiva™ C Series: this package enables the integration of micro-ROS in a exas Instruments Tiva™ C Series.",
```
Comment (Acuadros95) [inline scripts/component:42]:
```suggestion
        "description": "micro-ROS app for TI Tiva™ C Series: this package enables the integration of micro-ROS in Texas Instruments Tiva™ C Series.",
```

File: scripts/component [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import sys
+from textwrap import wrap
+import os
+
+MAX_CHARACTERS = 80
+
+COMPONENTS = [
+    {
+        "name": "renesas_e2_studio",
+        "description": "micro-ROS component for Renesas e2 studio and RA6M5: this package enables the integration of micro-ROS in Renesas e2 studio and RA6M5 MCU family.",
```
Comment (Acuadros95) [inline scripts/component:12]:
```suggestion
        "description": "micro-ROS component for Renesas e2 studio and RA6M5: this package enables the integration of micro-ROS in Renesas e2 studio for RA6M5 and RA6T2 MCUs.",
```

File: scripts/component [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+def print_help(extended = False):
+    if extended:
+        print("Usage: python3 component.py <component_name>")
+        print("")
+        for component in COMPONENTS:
+            description = "\n\t".join(wrap(component["description"], MAX_CHARACTERS))
+            print("[{}] \n\t{}\n".format(component["name"], description))
+    else:
+        print("Usage: python3 component.py <component_name>")
```
Comment (Acuadros95) [inline scripts/component:75]:
```suggestion
```

Is this a typo?

File: scripts/component [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        print("Available components: {}".format(", ".join([c["name"] for c in COMPONENTS])))
+
+# No arguments
+if len(sys.argv) < 2:
+    print_help()
+    sys.exit(0)
+
+# User asks for help
+if "-h" in sys.argv or "--help" in sys.argv:
+    print_help(True)
```
Comment (Acuadros95) [inline scripts/component:85]: Should we add a specific help command? like 

` ros2 run micro_ros_setup component arduino --help`

which returns:
```bash
[arduino] 
	Arduino IDE & CLI libraries for micro-ROS: this package enables the integration
	of micro-ROS in the Arduino IDE for some hardware platforms.

        For usage instructions visit: https://github.com/micro-ROS/micro_ros_espidf_component#readme
```

**Comment by pablogs9** [comment]
@Acuadros95 comments solved

**Review by Acuadros95 — APPROVED** [review]

**Pull request merged**